src/runtime/getPositions.ts

```diff
@@ -71,19 +73,23 @@ async function getBaseTokensInfo(
     tokensInfo[tokenId] = {
       ...tokenInfo,
       priceUsd: toSerializedDecimalNumber(tokenInfo.priceUsd ?? 0),
+      // We don't have this info here but it's not yet needed for base tokens anyway
+      balance: toDecimalNumber(0n, 0),
+      totalSupply: toDecimalNumber(0n, 0),
     }
   }
   return tokensInfo
 }
 
 async function getERC20TokenInfo(
-  address: Address,
+  contractAddress: Address,
   networkId: NetworkId,
+  address?: Address,
```

Comment:
i had to read through all the function code to understand what is this `address`. perhaps `userAddress` may be more meaningful and could've saved me from that 😅